snakemake/Snakefile

```diff
@@ -32,7 +32,7 @@ rule run_reciprocal_blast:
         eval = config.get("e_value_cutoff"),
         pident = config.get("minimum_percent_identity")
     shell:
-        "BackBLAST_core.py --gene_cluster {params.query_genes} --query_proteome {params.query_genome_orfs} --subject_proteome {input} "
+        "BackBLAST search --gene_cluster {params.query_genes} --query_proteome {params.query_genome_orfs} --subject_proteome {input} "
```

Comment:
Calling utility scripts now looks like this -- you call them through the BackBLAST entry script. I think this looks like a cleaner way to work with BackBLAST -- agreed?